cabal.project

```diff
@@ -174,16 +174,18 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-ledger-specs
-  tag: 7a24a6e752a24ccbca30a0f16af8fd868d3b6933
-  --sha256: 1xss7a3q9yznb1qq5rm8w2j1jsa5rmr96pnplrg1ixy55jqsdhly
+  tag: e8f19bcc9c8f405131cb95ca6ada26b2b4eac638
```

Comment:
I updated this dep, and then I updated all the rest to match what was in the `cardano-ledger-specs` repository's `cabal.project`. I was getting some build errors when entering the `nix-shell` until I did that.